internal/api/api.go

```diff
@@ -99,17 +102,36 @@ func (api *idler) Idle(w http.ResponseWriter, r *http.Request, ps httprouter.Par
 	w.WriteHeader(http.StatusOK)
 }
 
+func respondWithError(w http.ResponseWriter, status int, err error) {
+	log.Error(err)
+	w.WriteHeader(status)
+	w.Write([]byte(fmt.Sprintf("{\"error\": \"%s\"}", err)))
+}
+
 func (api *idler) UnIdle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+
 	openShiftAPI, openShiftBearerToken, err := api.getURLAndToken(r)
 	if err != nil {
-		log.Error(err)
-		w.WriteHeader(http.StatusBadRequest)
-		w.Write([]byte(fmt.Sprintf("{\"error\": \"%s\"}", err)))
+		respondWithError(w, http.StatusBadRequest, err)
+		return
+	}
+
+	ns := ps.ByName("namespace")
```

Comment:
this returns an empty string if no matches are found. Does it make sense to check for that here?